### ISSUE TITLE
Can anyone share some your results (eg：accuray）on some public datasets?

### ISSUE THREAD

**Issue by AnddyWang** [MaybeShewill-CV/CRNN_Tensorflow#94]
Can anyone share some your results (eg：accuray）on some public datasets?

**Comment by AnddyWang** [comment]
![Uploading 1528365044.png…]()
Is this normal?

**Comment by AnddyWang** [comment]
![1528365044](https://user-images.githubusercontent.com/19400072/41092692-0c8b978c-6a7c-11e8-8cc1-098009837fb6.png)

**Comment by baduy9x** [comment]
Same here, I have trained the model on my vietnamese dataset, but I have the problem with the loss. In the beginning, the loss decrease as usual, but it reaches a threshold and cannot decrease more. I don't know why, but the same code on Pytorch run very fast and the loss converge to 0 in few epoches. The decoded output of this model also very bad, is that a bug here?? @MaybeShewill-CV

**Comment by mhasnat** [comment]
Same problem in my case ... tried to ovecome the problem by changing the parameters of the config file ... no success yet.

**Comment by MaybeShewill-CV** [comment]
@mhasnat The repo has been updated you may test again:)
